src/NAR/RuntimeSystem.java

```diff
@@ -0,0 +1,73 @@
+package NAR;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class RuntimeSystem {
+	
+	private static RuntimeSystem Instance = new RuntimeSystem();
+	RuntimeSystem() {}
```

Comment:
you need to mark the constructor as private. As of now without the private keyword any other class in the same package as the RuntimeSystem class can create a RuntimeSystem object